www/index.html

```diff
@@ -694,6 +694,12 @@ <h3 data-i18n="configure-expert-settings-title">Expert settings</h3>
                                         <span data-i18n="configure-expert-bypassappcache"><b>Bypass AppCache</b> (<i>disables offline use of PWA!</i>)</span>
                                     </label>
                                 </div>
+                                <div class="checkbox" id="enableSourceVerificationCheckBox">
+                                    <label data-i18n-tip="configure-expert-enaable-source-verification" title="Warning: A few zim files might run malicious programs in your browser. This can however be prevented by using jquery mode, kiwix will automatically switch to jquery mode if you  mention you donot trust the source. Disabling this option will result in using serviceworker mode by default, if available." >
+                                        <input type="checkbox" name="disableFileVerification" id="enableSourceVerification" >
+                                        <span data-i18n="configure-expert-disable-source-verification"><strong>Enable source verification of new files</strong> (once checked, kiwix will ask to verify file source before opening.)</span>
```

Comment:
Another suggestion for clarity:

```suggestion
                                        <span data-i18n="configure-expert-disable-source-verification"><strong>Enable source verification of new files</strong> (<i>recommended</i>: you will only be prompted the first time you open a ZIM)</span>
```